docs/advanced_topics/documents/custom_document_model.md

```diff
@@ -0,0 +1,68 @@
+(custom-document-model)=
```

Comment:
Can this reference be updated to lower_snake_case

This may mean updating other links that use this, however we are trying to standardise on the reference names being one format. 

https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#reference-links

Reply:
my bad, did not read that section, i'll push the update, one more thing will this conversion (rst -> md) be done for all of docs/*